validation/chOpenApiValidator_test.go

```diff
@@ -0,0 +1,220 @@
+package validation
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/getkin/kin-openapi/openapi3filter"
+	"github.com/getkin/kin-openapi/routers"
+	router "github.com/getkin/kin-openapi/routers/gorillamux"
+	. "github.com/smartystreets/goconvey/convey"
+	"net/http/httptest"
+	"testing"
+)
+
+const (
```

Comment:
Is it possible to load this from a file ?

Reply:
Changed to just be a small dummy data request.